myning/chapters/enter_graveyard.py

```diff
@@ -82,8 +81,9 @@ def revive(player: Player, ally: Character):
         FileManager.multi_save(player, ally)
 
 
-def soul_cost(ghost_count: int):
-    return int(fibonacci(ghost_count + 1))
+@cache
```

Comment:
Why cache?

Reply:
I guess it doesn't hurt actually. It just seems like a cheap calculation to cache. I approve and I will now migrate to GitHub :)

Reply:
You're right. I cached it when it used to be recursive. I'll remove it.